op-challenger/game/fault/solver/game_solver_test.go

```diff
@@ -84,18 +101,109 @@ func TestCalculateNextActions(t *testing.T) {
 					Attack(maliciousStateHash)
 			},
 		},
+		{
```

Comment:
Let's add more one more test case for freeloaders disputing root. This test case fails on the other feature branch.
```go
			builder.Seq().
					ExpectAttack().            // Honest response to invalid root
					Attack(common.Hash{0xaa}). // freeloader
					ExpectAttack()             // Honest response to freeloader
```